mockafka/aiokafka/aiokafka_producer.py

```diff
@@ -28,8 +31,18 @@ class FakeAIOKafkaProducer:
     def __init__(self, *args, **kwargs) -> None:
         self.kafka = KafkaStore()
 
+    def _translate_header_to_internal_format(self, headers: Sequence[Tuple[str, bytes]]) -> dict:
+        _header_dict: dict = defaultdict(dict)
```

Comment:
What is the precise type of this expected to be? Here it looks like it's `dict[?, dict[?, ?]]` yet later on in the loop the assignments seem to be bytes, suggesting it should be `dict[str, bytes]`.

Minor: this also reads a little odd that a local variable starts with an underscore.